httpbp/headers.go

```diff
@@ -24,6 +24,9 @@ const (
 	// the HTTP request headers.
 	ParentIDHeader = "X-Parent"
 
+	// RetryAfterHeader is the standard "Retry-After" header key.
+	RetryAfterHeader = "Retry-After"
```

Comment:
This is a weird place to put it. This is a standard HTTP header, not tracing related or something baseplate defined. We probably should put it out of this const block.